contact-us.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Contact Us</title>
+</head>
+
+<body>
+
+  <h1>Contact Us</h1>
+
+  <p>Feel free to reach out to us!</p>
+
+  <form action="mailto:your@email.com" method="post" enctype="text/plain">
+    Name:<br>
+    <input type="text" name="name" required><br>
+    Email:<br>
+    <input type="email" name="email" required><br>
```

Comment:
please remove required attribute and add id as "email"

Reply:
Please remove required and instead some value can be added next to the name instead of required as ( type name value)